grakn-graph/src/main/java/ai/grakn/graph/internal/computer/GraknComputerImpl.java

```diff
@@ -118,4 +136,22 @@ protected GraphComputer getGraphComputer() {
         return graph.compute(this.graphComputerClass);
     }
 
+    private void applyFilters(Set<LabelId> types, boolean includesShortcut) {
+        if (types == null || types.isEmpty()) return;
```

Comment:
In this case, we ignore `includesShortcut`. Is that a bug?
Also tag `types` as `@Nullable`! This is my new vendetta :D